web/src/main.tsx

```diff
@@ -213,29 +216,29 @@ const router = createBrowserRouter([
       },
       {
         path: '/map',
-        element: <Navigate to="/map/72h" replace />,
+        element: <Navigate to="/map/hourly/72h" replace />,
```

Comment:
Why do we need the `72h` in the URL if it's always that? 🤔

Reply:
Oli wants both in there, so both are in there...

Reply:
I think @madsnedergaard is asking about the `72h`, not the fact that `hourly` is present?

Reply:
I think he means why do we need both, but the reason is that if we need to supports but monthly all and yearly all without saying monthly all and yearly all in the url we need another limiter, this really increases the complexity for the URL syncing though so I’m honestly not sure it’s worth it for just clean URLs, but it’s done now.

before we could just read the url parameter and check if it’s valid, now we need to read both of them and do a bunch of conditional logic as well.

Reply:
If that's the question then indeed we need both resolution and range in the URL because now the same range can have multiple resolutions (and vice-versa). The URL encoding could have been made by concatenating both (e.g. `monthly_all`), but I believe we benefit in the long-run from a cleaner encoding that clearly separates the two parameters, now that one can't be inferred from the other.

Reply:
Makes sense, it just seemed silly with `/hourly/72h` if hourly is only 72h, but I talked with Viktor and see now the future path and why it makes sense :)